### ISSUE TITLE
Zod models are undefined on runtime for client components

### ISSUE THREAD

**Issue by eelior** [zenstackhq/zenstack#861]
**Description and expected behavior**
After following the instructions in the [Form Validation](https://zenstack.dev/blog/form-validation) blog post, I attempted to use the generated zod models. They seemed to be working correctly at first, but then I encountered a runtime error.

**Screenshots**

![image](https://github.com/zenstackhq/zenstack/assets/59517348/c98dc541-d993-4e38-9dcb-2c8920fe0f53)
```
(alias) const ClientCreateSchema: z.ZodObject<{
    id: z.ZodOptional<z.ZodString>;
    createdAt: z.ZodOptional<z.ZodDate>;
    updatedAt: z.ZodOptional<z.ZodDate>;
    email: z.ZodOptional<z.ZodNullable<z.ZodString>>;
    phone: z.ZodOptional<z.ZodNullable<z.ZodString>>;
    firstName: z.ZodString;
    lastName: z.ZodString;
}, "strip", z.ZodTypeAny, {
    firstName: string;
    lastName: string;
    id?: string | undefined;
    createdAt?: Date | undefined;
    updatedAt?: Date | undefined;
    email?: string | null | undefined;
    phone?: string | null | undefined;
}, {
    firstName: string;
    lastName: string;
    ... 4 more ...;
    phone?: string | null | undefined;
}>
```
<img width="979" alt="image" src="https://github.com/zenstackhq/zenstack/assets/59517348/830bd98f-4968-468a-8f24-3a4c166d6ad2">

```
Unhandled Runtime Error
TypeError: Cannot read properties of undefined (reading 'ClientCreateSchema')
```

**Environment:**

-   ZenStack version: 1.1.1
-   Prisma version: 5.1.1
-   Database type: sqlite

**Additional context**
I have searched everywhere but couldn't find a similar issue. would greatly appreciate any help or direction in debugging it. 🙏🏽

**Comment by ymc9** [comment]
Hi @eelior , thanks for reporting this! Do you have a repro project that we can look into?

**Comment by eelior** [comment]
Here you go:
https://github.com/eelior/zenstack-zod-model-undefined

While creating this repro I noticed this is happening only in **client components**, see 2 lasts commits. 
Updating issue title accordingly.

**Comment by ymc9** [comment]
Hi @eelior , sorry for the latest response.

It seems that Next13's app dir has stricter rules about the way modules are exported and required. I'm making a fix, and it'll be included in the v1.5.0 release this week. Thanks for reporting the issue!
